Assets/Scripts/PowerUps/PowerUpProcessor.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Reflection;
+using UnityEngine;
+
+public static class PowerUpProcessor
+{
+    private static AbstractPowerUp apu;
```

Comment:
I feel that PowerUpProcessor's functionality can be merged into PowerUpManager. Is there a particular reason for splitting them into two?